### DISCUSSION TITLE
How to remove "github-project-automation"?

### DISCUSSION THREAD

**Discussion by jsh9** [jsh9/pydoclint#172] [General]
Hi @real-yfprojects , do you know how I can remove "github-project-automation"?  Specifically, in every issue under this project (such as this: https://github.com/jsh9/pydoclint/issues/169), there are 2 links:

<img width="1105" alt="image" src="https://github.com/user-attachments/assets/805f41fc-52fd-4938-b48c-e316b60a40cc">

Thanks!

**Comment by real-yfprojects** [comment]
You cannot remove it, I don't why github shows this project for everyone. Though I changed the visibility of the project to private. Is it fixed now?
